### DISCUSSION TITLE
Training always producing same accuracy for train and val sets

### DISCUSSION THREAD

**Discussion by SasikiranJ** [stellargraph/stellargraph#1836] [Q&A]
When i try to train model on train and validation sets . It is always producing same accuracy over number of epochs. What i have to do in that case?

**Comment by adocherty** [comment]
Hi, without more details of your problem or code it is difficult to offer any suggestions.  Often when this happens, checking the network you have created is calculating what you expect is a good first step.  If the network appears to be correct, try adjusting the training parameters to see if this improves things.

I often find that trying the same problem with a standard (non-graph) neural network first can help provide a baseline performance and see if there are any issues in other parts of your code.

I hope this helps!
